.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    [
+      "babel-plugin-styled-components",
+      { "ssr": true, "displayName": true, "preprocess": false }
+    ]
+  ]
```

Comment:
Hey Walter, what did you do here? Do you need that file?

Reply:
Alex said it is needed. I didn´t  have the time yet  to test it, that is why I just trust him